pallets/runtime/tests/identity_test.rs

```diff
@@ -60,18 +60,35 @@ fn fetch_systematic_cdd(target: IdentityId) -> Option<IdentityClaim> {
 // =======================================
 
 #[test]
+fn add_claims_batch_test() {
+    ExtBuilder::default()
+        .existential_deposit(1_000)
+        .monied(true)
+        .cdd_providers(vec![
+            AccountKeyring::Eve.public(),
+            AccountKeyring::Ferdie.public(),
+        ])
+        .build()
+        .execute_with(|| add_claims_batch());
+}
+
 fn add_claims_batch() {
-    ExtBuilder::default().build().execute_with(|| {
+    
         let _owner_did = register_keyring_account(AccountKeyring::Alice).unwrap();
         let _issuer_did = register_keyring_account(AccountKeyring::Bob).unwrap();
         let claim_issuer_did = register_keyring_account(AccountKeyring::Charlie).unwrap();
-        let claim_issuer = AccountKeyring::Charlie.public();
+        let _claim_issuer = AccountKeyring::Charlie.public();
+        let cdd_claim_issuer = AccountKeyring::Eve.public();
+
         let scope = Scope::from(0);
 
+        // Check that CDD Issuer now has an did.
+        let cdd_claim_id = get_identity_id(AccountKeyring::Eve).unwrap();
```

Comment:
Eve might not have a DID at this point, which is probably why the `unwrap` panics.

Reply:
ohh, I would try adding it then and get back to you.

Reply:
Sorry, the error is further down.